tests/js/spec/views/dashboardsV2/gridLayout/dashboard.spec.tsx

```diff
@@ -225,4 +222,43 @@ describe('Dashboards > Dashboard', () => {
       expect(await screen.findByText('Matching Issue Owners Rule')).toBeInTheDocument();
     });
   });
+
+  describe('Edit mode', () => {
+    const widgets = [newWidget];
+    const mount = (dashboard, mockedOrg = initialData.organization) => {
+      const getDashboardComponent = () => (
+        <Dashboard
+          paramDashboardId="1"
+          dashboard={dashboard}
+          organization={mockedOrg}
+          isEditing
+          onUpdate={newWidgets => {
+            widgets.splice(0, widgets.length, ...newWidgets);
+          }}
+          handleUpdateWidgetList={() => undefined}
+          handleAddCustomWidget={() => undefined}
+          onSetWidgetToBeUpdated={() => undefined}
+          router={initialData.router}
```

Comment:
What happens if `mockedOrg` is passed in? Can we make the assumption that `initialData.router` applies here as well?

Reply:
Good point, probably not. I actually don't think I need the org as an arg so I've just removed the arg for now and kept the organization as `initialData.organization` to avoid this issue. Thanks!